testcases.py

```diff
@@ -924,10 +924,17 @@ def check(self) -> TestResult:
         client = {0: 0, 1: 0}
         server = {0: 0, 1: 0}
         try:
+
+            def _get_key_phase(pkt) -> int:
```

Comment:
Please add a comment saying that older and newer versions (if possible, specify the version number) of Wireshark output the key phase bit differently.